test/fixtures/api-client/locations_free_spaces.findAll.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
Are there any model changes that need to happen alongside this addition?

Reply:
Model changes for this have been added in the service PR - #948